### DISCUSSION TITLE
CMD窗口运行了alist之后不能关闭，关闭后管理网页打不开，不关则能正常运行。

### DISCUSSION THREAD

**Discussion by simon333888** [AlistGo/alist#2031] [Invalid question]
遇到一个很有意思的问题，CMD窗口运行了alist之后不能关闭，关闭后管理网页打不开，不关则能正常运行。不知道是哪里出了问题。
![image](https://user-images.githubusercontent.com/40132809/196012885-af01b984-d255-4024-9758-6c9c8ddc9ead.png)
![image](https://user-images.githubusercontent.com/40132809/196012900-fbac5f65-2332-4828-b410-3ea33b8f5f18.png)

**Comment by EwdRe** [comment]
就是这样的，关了程序就不运行了，可以参考一下windows的进程守护

**Reply by jilefo** [reply]
https://www.52pojie.cn/thread-1713978-1-1.html 参考这个帖子

**Comment by anwen-anyi** [comment]
https://www.bilibili.com/video/BV1kP411J7YN/

**Comment by haorwen** [comment]
不然嘞，关了之后进程不就停了吗。。

**Comment by simon333888** [comment]
我已经解决了，原来这个窗口需要隐藏起来而不是关掉。




wlsimon
***@***.***



&nbsp;





------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "alist-org/alist"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年11月7日(星期一) 晚上7:54
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [alist-org/alist] CMD窗口运行了alist之后不能关闭，关闭后管理网页打不开，不关则能正常运行。 (Discussion #2031)





 
不然嘞，关了之后进程不就停了吗。。
